release_dashboard/lib/widgets/create_release_substeps.dart

```diff
@@ -56,53 +76,61 @@ class CreateReleaseSubstepsState extends State<CreateReleaseSubsteps> {
           index: 0,
           setReleaseData: setReleaseData,
           hintText: 'The candidate branch the release will be based on.',
+          changeIsInputValid: changeIsEachInputValid,
         ),
         CheckboxListTileDropdown(
           index: 1,
           releaseData: releaseData,
           setReleaseData: setReleaseData,
           options: const <String>['dev', 'beta', 'stable'],
+          changeIsDropdownValid: changeIsEachInputValid,
         ),
         InputAsSubstep(
           index: 2,
           setReleaseData: setReleaseData,
           hintText: "Git remote of the Conductor user's Framework repository mirror.",
+          changeIsInputValid: changeIsEachInputValid,
         ),
         InputAsSubstep(
           index: 3,
           setReleaseData: setReleaseData,
           hintText: "Git remote of the Conductor user's Engine repository mirror.",
+          changeIsInputValid: changeIsEachInputValid,
         ),
         InputAsSubstep(
           index: 4,
           setReleaseData: setReleaseData,
           hintText: 'Engine cherrypick hashes to be applied. Multiple hashes delimited by a comma, no spaces.',
+          changeIsInputValid: changeIsEachInputValid,
         ),
         InputAsSubstep(
           index: 5,
           setReleaseData: setReleaseData,
           hintText: 'Framework cherrypick hashes to be applied. Multiple hashes delimited by a comma, no spaces.',
+          changeIsInputValid: changeIsEachInputValid,
         ),
         InputAsSubstep(
           index: 6,
           setReleaseData: setReleaseData,
           hintText: 'New Dart revision to cherrypick.',
+          changeIsInputValid: changeIsEachInputValid,
         ),
         CheckboxListTileDropdown(
           index: 7,
           releaseData: releaseData,
           setReleaseData: setReleaseData,
           options: const <String>['y', 'z', 'm', 'n'],
+          changeIsDropdownValid: changeIsEachInputValid,
         ),
         const SizedBox(height: 20.0),
         Center(
-          // TODO(Yugue): Add regex validation for each parameter input
-          // before Continue button is enabled, https://github.com/flutter/flutter/issues/91925.
           child: ElevatedButton(
             key: const Key('step1continue'),
-            onPressed: () {
-              widget.nextStep();
-            },
+            onPressed: isEachInputValid.contains(false) // Continue button is disabled if any of the inputs are invalid
```

Comment:
nit: rather than

```
() {
  widget.nextStep(0;
},
```

You can do a tear-off: `onPressed: isEachInputValid.contains(false) ? null : widget.nextStep,`